### DISCUSSION TITLE
Option to make the pop-out race control messages window stay always on top

### DISCUSSION THREAD

**Discussion by RCD800** [f1multiviewer/issue-tracker#66] [Feature requests]
Would be nice to have the pop-out window on top, so i could have it on top of discord and continue chatting.
Also if not suggested already, an option to remove its borders like it is on the videos.

**Comment by SimplyBlu** [comment]
Would be nice to take this idea plus #72 (launch/run race control independent of live timing) for a better race control message experience.

**Reply by mozjag** [reply]
Same for Team Radio and the Live Timing Track Map, please!

**Comment by NeuroDawg** [comment]
I would love to see a separate window/stream for race control messages. Ideally it would be like the track map - borderless window and transparent and always on top. I'd love to be able to place this over the lower section of my main race window.

**Reply by mozjag** [reply]
In the (Replay) Live Timing window's settings menu you can find "Open Race Control Messages" which opens a regular window with title bar and without transparency etc., but at least it lets you place it where you want. Hopefully things like borderless, transparency, always on top etc. will come soon.
